src/main/java/Duke.java

```diff
@@ -1,10 +1,129 @@
+import java.util.Scanner;
+
 public class Duke {
+
+    public static int taskCount = 1;
+    public static Task[] tasks = new Task[101];
+
+    public static void goodbye() {
+        System.out.println("\n" +
+                "░██████╗░░█████╗░░█████╗░██████╗░██████╗░██╗░░░██╗███████╗\n" +
+                "██╔════╝░██╔══██╗██╔══██╗██╔══██╗██╔══██╗╚██╗░██╔╝██╔════╝\n" +
+                "██║░░██╗░██║░░██║██║░░██║██║░░██║██████╦╝░╚████╔╝░█████╗░░\n" +
+                "██║░░╚██╗██║░░██║██║░░██║██║░░██║██╔══██╗░░╚██╔╝░░██╔══╝░░\n" +
+                "╚██████╔╝╚█████╔╝╚█████╔╝██████╔╝██████╦╝░░░██║░░░███████╗\n" +
+                "░╚═════╝░░╚════╝░░╚════╝░╚═════╝░╚═════╝░░░░╚═╝░░░╚══════╝\n" +
+                "Hope to see you again soon!");
+    }
+
+    public static void welcome() {
+        System.out.println("\n" +
+                "██╗░░██╗███████╗██╗░░░░░██╗░░░░░░█████╗░\n" +
+                "██║░░██║██╔════╝██║░░░░░██║░░░░░██╔══██╗\n" +
+                "███████║█████╗░░██║░░░░░██║░░░░░██║░░██║\n" +
+                "██╔══██║██╔══╝░░██║░░░░░██║░░░░░██║░░██║\n" +
+                "██║░░██║███████╗███████╗███████╗╚█████╔╝\n" +
+                "╚═╝░░╚═╝╚══════╝╚══════╝╚══════╝░╚════╝░\n" +
+                "from\n" + "\n" +
+                "██████╗░██╗░░░██╗██╗░░██╗███████╗\n" +
+                "██╔══██╗██║░░░██║██║░██╔╝██╔════╝\n" +
+                "██║░░██║██║░░░██║█████═╝░█████╗░░\n" +
+                "██║░░██║██║░░░██║██╔═██╗░██╔══╝░░\n" +
+                "██████╔╝╚██████╔╝██║░╚██╗███████╗\n" +
+                "╚═════╝░░╚═════╝░╚═╝░░╚═╝╚══════╝\n");
+        System.out.println("Hello! I'm Duke\nWhat can I do for you?\n");
+    }
+    public static void showList() {
+        if (taskCount > 1) {
+            System.out.println("Here are the tasks in your list:");
+            for (int i = 1; i < taskCount; i++) {
+                System.out.println(i + ". " + tasks[i].getTaskType() + " " + tasks[i].getStatusIcon()
+                        + tasks[i].getDescription());
+            }
+        } else {
+            System.out.println("Please input a task.");
+        }
+    }
+
+    public static void done(String taskDone) {
+        int taskIndex = Integer.parseInt(taskDone);
+        if (taskIndex >= taskCount || taskIndex < 0){
```

Comment:
Should this be extracted out as something like if(hasTask()) for better readability?